Framework/Geometry/src/Instrument/RectangularDetector.cpp

```diff
@@ -45,7 +46,7 @@ RectangularDetector::RectangularDetector()
       m_minDetId(0), m_maxDetId(0) {
 
   init();
-  setGeometryHandler(new BitmapGeometryHandler(this));
+  setGeometryHandler(new GeometryHandler(this));
```

Comment:
I can see that this was done originally, but would be better to move away from explicit new and pointers for this API. Since `ObjComponent` should die off, maybe this wouldn't be worth fixing here?

Reply:
`RectangularDetector` and other things like this should die off so it can probably be fixed later on in the release branch.